Readme.md

```diff
@@ -3,37 +3,28 @@
-<!-- default file list end -->
-# ASPxGridView - Batch Edit - How to change a cell value based on another cell value
 
+# GridView for Web Forms - How to change a cell value based on another cell value in batch edit mode
 
-<p>The sample illustrates how to change the Price column value when the Percentage column is changed and vice versa. <br><br>ASPxClientGridView provides the <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebScriptsASPxClientGridView_batchEditApitopic">ASPxClientGridView.batchEditApi</a> property to work with grid in batch mode on the client side. In the <a href="https://documentation.devexpress.com/#AspNet/clsDevExpressWebScriptsASPxClientGridViewBatchEditApitopic">ASPxClientGridViewBatchEditApi</a> class, there are two methods to get and set a cell value respectfully: <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebScriptsASPxClientGridViewBatchEditApi_GetCellValuetopic">ASPxClientGridViewBatchEditApi.GetCellValue</a>, <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebScriptsASPxClientGridViewBatchEditApi_SetCellValuetopic(U0XPfw)">ASPxClientGridViewBatchEditApi.SetCellValue(Int32,String,Object)</a>.<br><br>In the <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebScriptsASPxClientGridView_BatchEditEndEditingtopic">ASPxClientGridView.BatchEditEndEditing</a> event handler, the GetCellValue method returns a correct cell value when a cell is not in <strong>editing mode</strong>. If a cell is in <strong>editing mode</strong> and a value is changed, the GetCellValue method returns the previous value, not a new entered value. To get a new entered value that is still in <strong>editing mode</strong>, it's necessary to use the <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebScriptsASPxClientGridViewBatchEditEndEditingEventArgs_rowValuestopic">ASPxClientGridViewBatchEditEndEditingEventArgs.rowValues</a> property. <br><br>In the attached sample, a focused column is saved in the <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebScriptsASPxClientGridView_BatchEditStartEditingtopic">ASPxClientGridView.BatchEditStartEditing</a> event handler:</p>
+The example demonstrates how to change a column value (Price) when another column (Percentage) is changed and vice versa.
 
+![Grid View - Changed Values](grid-batch-linked-values.png)
 
-```js
-var fieldName;
-function OnBatchEditStartEditing(s, e) {
-    fieldName = e.focusedColumn.fieldName;
-}
- 
+[ASPxClientGridView](https://docs.devexpress.com/AspNet/js-ASPxClientGridView) class implements the [batchEditApi](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.batchEditApi) property that exposes API to work with grid in batch mode on the client side.
 
-```
+When value editing is over, the [BatchEditEndEditing](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.BatchEditEndEditing) event is raised. In the event handler, you can get a cell value in the following ways:
```

Comment:
`When value editing is over`

I'd like to specify _who_ edits the value if possible. Is it a user or developer?

Reply:
```suggestion
When a user finishes value editing, the [BatchEditEndEditing](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.BatchEditEndEditing) event is raised. In the event handler, you can get a cell value in the following ways:
```

Reply:
How about the following?

`After a user edits the value, the [BatchEditEndEditing](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.BatchEditEndEditing) event is raised. In the event handler, you can get a cell value in the following ways:`

Reply:
```suggestion
After a user edits the value, the [BatchEditEndEditing](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.BatchEditEndEditing) event is raised. In the event handler, you can get a cell value in the following ways:
```